apps/console/src/features/claims/components/edit/local-claim/edit-basic-details-local-claims.tsx

```diff
@@ -549,7 +549,7 @@ export const EditBasicDetailsLocalClaims: FunctionComponent<EditBasicDetailsLoca
                                 value={ claim?.regEx }
                                 ref={ regExField }
                                 data-testid={ `${ testId }-form-regex-input` }
-                                maxLength={ 50 }
+                                maxLength={ 255 }
```

Comment:
shall we move this to a const file?

Reply:
Improved with cf31631b8a4bf6e56861ee08268d2ec9396549e9.